Java/small_game/Game.java

```diff
@@ -0,0 +1,122 @@
+/*
+        Simple program based on basics of java
+        i.e. Input/Output and simple decision making
+        with very well formatted Output
+*/
+import java.util.Scanner;
+public class Game
+{
+    public static void main(String args[])
+    {
+        Scanner in=new Scanner(System.in);
+        System.out.println("==========================================================");
+        System.out.println("welcome to WHO WHANT TO BE TOKEN KING");
+        System.out.println("==========================================================");
+        System.out.print("enter your name : ");
+        String name;
+        name=in.nextLine(); // Name of Player is inputted here
+        System.out.println("==========================================================");
+        System.out.println();
+        System.out.println();
+        System.out.println();
+        int s=0;
+        System.out.println("%%%%%%%%Instruction are as follows%%%%%%%%%");// Instruction for Playing
+        System.out.println("1.You should not do any cheating");
+        System.out.println("2.Five question will be asked answer them by giving its respective  letter");
+        System.out.println("3.If you give all 5 questions correctly u will be given ur token back");

```

Comment:
Correct the "u" here and all other instances ahead

Reply:
Okay i will incorporate this changes